macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -940,7 +940,7 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 
         UserDefaultsWrapper<Any>.clearRemovedKeys()
 
-        vpnSubscriptionEventHandler?.registerForSubscriptionAccountManagerEvents()
+        vpnSubscriptionEventHandler?.startMonitoring()
```

Comment:
Ok so this is a big part of the fix.  We were previously initializing some stuff in `vpnSubscriptionEventHandler`'s initializer - but that was triggered BEFORE the subscription was fully set up, so we were getting tons of failures.

I now start everything here (including the call I replaced in this line).

Reply:
Nice one!